features/app-tracker-protection.json

```diff
@@ -49,6 +49,7 @@
                     "domain": "dpm.demdex.net",
                     "packageNames": [
                         {
+                            "packageName": "com.bestbuy.android",
```

Comment:
These actually have to be separate objects - as per examples below.

Reply:
Ooops -- too excited about vacation, sorry 😆